.sass-lint.yml

```diff
@@ -0,0 +1,220 @@
+linters:

```

Comment:
Yup, please do. We already have `.scss-lint.yml` and you can just keep the `.sass-lint.yml` locally on your machine